### ISSUE TITLE
Skipping rows for header

### ISSUE THREAD

**Issue by armaankohli91** [jcustenborder/kafka-connect-spooldir#164]
Hi

I understand csv.first.row.as.header marks the first row as header. But in a scenario where the csv has some file headers that I want to skip, I couldn't find any way to reach the actual header data.
I wish to keep schema.generation.enabled as true.
Though we have csv.skip.lines, it seems to be skipping the rows for reaching the data actually (not the header).

How do I skip n number of rows to identify the header first?

Eg. csv:
Student details
Id, Name, Age
1,abc,23
2,xyz,21

Where "Student details" is a text present on first row, and needs to be skipped. Header starts from second row.

**Comment by jcustenborder** [comment]
Hi @armaankohli91, 

Off hand I'm not completely sure. This connector wraps opencsv for the csv parsing functionality. The value you specify for `csv.skip.lines` gets passed to [this method](http://opencsv.sourceforge.net/apidocs/com/opencsv/CSVReaderBuilder.html#withSkipLines-int-). Can you run the connector with trace logging enabled and upload the logs? 

```
log4j.logger.com.github.jcustenborder.kafka.connect.spooldir=TRACE
```

**Comment by armaankohli91** [comment]
Great. Fix #166 should be able to add this feature.
